AISKU/src/AISku.ts

```diff
@@ -259,8 +257,31 @@
                     _self.emptyQueue();
                     _self.pollInternalLogs();
                     _self.addHousekeepingBeforeUnload(_self);
+
+                    _addUnloadHook(onConfigChange(_config, () => {
+                        if (!_throttleMgr){ //&& cfgHandler.cfg.disabled
+                            _throttleMgr = new ThrottleMgr(_core);
+                        }
+                        //  && !_config.throttleMgrCfg.disabled
+                        if (_throttleMgr && !_throttleMgr.isReady() && _config.extensionConfig && _config.extensionConfig[_cfgSyncPlugin.identifier]) {
```

Comment:
Once you move the `_throttleMgr = new ThrottleMgr(_core)`;  outside of the `onConfigChange` this will either self resolve  or we can just dismiss the alert as it would then be possible for the `_throttleMgr` to be null when the called is called.